tests_integration/README.md

```diff
@@ -0,0 +1,96 @@
+# Integration Tests
+This directory contains integration tests for the autotick-bot.
+The tests are run against actual GitHub repositories, and are used to verify that the
+bot works as expected in an environment closely resembling production.
+
+## Test Environment
+The integration tests operate in a testing environment consisting of three real GitHub entities:
+
+- [conda-forge-bot-staging](https://github.com/conda-forge-bot-staging) (organization) mimics the
+[conda-forge](https://github.com/conda-forge) organization and will contain a selection of test feedstocks
+(see below how we create them)
+- [regro-cf-autotick-bot-staging](https://github.com/regro-cf-autotick-bot-staging) (user) mimics the
+[regro-cf-autotick-bot](https://github.com/regro-cf-autotick-bot) account and is a test environment in which the bot
+will create forks of the conda-forge-bot-staging repositories
+- [regro-staging](https://github.com/regro-staging) (organization) (named after the [regro](https://github.com/regro)
+account) contains a special version of the [cf-graph-countyfair](https://github.com/regro/cf-graph-countyfair) which
+the bot uses during testing.
+
+## Test Cases Definition
+The integration tests are defined in the [definitions](definitions) directory. The following directory structure is
+used (using `pydantic` and `llvmdev` as example feedstocks):
+
+```text
+definitions/
+├── pydantic/
+│   ├── resources/
+│   │   ├── feedstock
+│   │   └── ... (entirely custom)
+│   ├── version_update.py
+│   ├── aarch_migration.py
+│   ├── some_other_test_case.py
```

Comment:
Where do these names come from? Do they have to match something in the bot code?

Reply:
clarified in ea48c90f7706ca6738648dfde813e3043fe5e34b